.tours/3---dynamic flows.tour

```diff
@@ -0,0 +1,95 @@
+                    "character": 1
+                },
+                "end": {
+                    "line": 13,
+                    "character": 1
+                }
+            }
+        },
+        {
+            "file": "es-space-mgmt/main/default/lwc/reservationHelper/reservationHelper.js",
+            "description": "`reservationHelper` LWC component acts a bridge between `reservationHelperForm` LWC flow screen component and `reservationHelperAura` component.",
+            "line": 1
+        },
+        {
+            "file": "es-space-mgmt/main/default/lwc/reservationHelper/reservationHelper.js",
+            "description": " The component uses [LMS](https://developer.salesforce.com/docs/component-library/bundle/lightning-message-service/documentation) to subscribe to flow status change event and customer tile selection event",
+            "line": 1,
+            "selection": {
+                "start": {
+                    "line": 4,
+                    "character": 1
+                },
+                "end": {
+                    "line": 11,
+                    "character": 1
+                }
+            }
+        },
+        {
+            "file": "es-space-mgmt/main/default/lwc/reservationHelperForm/reservationHelperForm.js-meta.xml",
+            "description": "`reservationHelperForm` LWC component is configured to work in flow screens. The component target config uses lightning__FlowScreen as a target. Read more about configuring LWC component for flow screen [here](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.use_config_for_flow_screens)",
+            "line": 9,
+            "selection": {
+                "start": {
+                    "line": 9,
+                    "character": 1
+                },
+                "end": {
+                    "line": 54,
+                    "character": 1
+                }
+            }
+        },
+        {
+            "file": "es-space-mgmt/main/default/lwc/reservationHelperForm/reservationHelperForm.js",
+            "description": "FlowNavigationNextEvent event advances flow to the next flow screen. You can read about ***FlowNavigationNextEvent*** [here](https://developer.salesforce.com/docs/component-library/bundle/lightning-flow-support/documentation)",
+            "line": 58,
+            "selection": {
+                "start": {
+                    "line": 58,
+                    "character": 1
+                },
+                "end": {
+                    "line": 59,
+                    "character": 1
+                }
+            }
+        },
+        {
+            "file": "es-space-mgmt/main/default/flows/createReservation.flow-meta.xml",
+            "description": "`Create Reservation` flow is a multi-screen flow with LWC, standard flow screen components, and apex action. This flow creates reservation records and updates the record status field.",
+            "line": 1
+        }
+    ]
+}
```

Comment:
Hmm. The architecture for this is a lil complex, as we talk about a lot of components here.. May be we can summarize the architecture in the first step of the tour, or maybe point to an image of the architecture.

Reply:
and we can start this tour by first talking about the flow, and then show how we invoke the flow. coz right now, we talk about all the components, and finally, come to the flow... By which point I have already seen many components and I am lost.

Reply:
added architecture diagrams as png. Hoping that helps. This will be a work in progress and hoping to refine it as the flows team gets us the dynamic flows feature for LWC.